src/extra/AutoUi/Collection/PersistentFilters.tsx

```diff
@@ -0,0 +1,184 @@
+// import { UnregisterCallback } from 'history';
+import * as React from 'react';
+import filter from 'lodash/filter';
+import qs from 'qs';
+import { JSONSchema } from '../../../components/Renderer/types';
+import {
+	Filters,
+	FilterSignature,
+	FiltersProps,
+	FiltersView,
+} from '../../../components/Filters';
+import {
+	createFilter,
+	createFullTextSearchFilter,
+	decodeFilter,
+	flattenSchema,
+	FULL_TEXT_SLUG,
+} from '../../../components/Filters/SchemaSieve';
+import { getFromLocalStorage, setToLocalStorage } from '../../../utils';
+import { History } from '../../../contexts/HistoryContext';
+
+export interface ListQueryStringFilterObject {
+	n: FilterSignature['field'];
+	o: FilterSignature['operator'];
+	v: FilterSignature['value'];
+}
+
+export const isListQueryStringFilterRule = (
```

Comment:
Do we still need to export this?
Also let's move isQueryStringFilterRuleset right below this one so that it's easier to spot the dependencies :)